cmake/compilerFlags.cmake

```diff
@@ -45,8 +45,10 @@ if ( MINGW OR UNIX OR MSYS ) # MINGW, Linux, APPLE, CYGWIN
                     set(SANITIZER_FLAGS "-fno-omit-frame-pointer -fsanitize=address")
                 endif()
             elseif( COMPILER_IS_CLANG )
-                if ( CMAKE_CXX_COMPILER_VERSION VERSION_GREATER 3.3 )
+                if ( CMAKE_CXX_COMPILER_VERSION VERSION_GREATER 4.9 )

```

Comment:
There seems to be an error here. These versions are referring only to clang, and 4.9 seems to be a GCC version.

Reply:
Ah, I think the check is valid.  It has something to do with the version of the C++ library or something.   The point of the change is to ensure that well can build ASAN on Ubuntu 14.04 with the default compiler (3.4), on Ubuntu 16.04 (3.8) and 18.04 (6.0).  This change enables that.  I believe the builds have passed on GitLab, although I don't know if we're requesting ASAN on GitLb.